Source/Scene/ScreenSpaceCameraController.js

```diff
@@ -465,7 +465,11 @@ define([
         var camera = scene.camera;
         var mode = scene.mode;
 
-        var pickedPosition = mode !== SceneMode.SCENE2D ? pickGlobe(object, startPosition, scratchPickCartesian) : camera.getPickRay(startPosition, scratchZoomPickRay).origin;
+        var pickedPosition;
```

Comment:
Same comment for build.xml and package.json below.  Perhaps your branch is just out of date?